command/kv_put.go

```diff
@@ -201,11 +201,14 @@ func (c *KVPutCommand) Run(args []string) int {
 		}
 		return 2
 	}
-	if secret == nil {
+
+	// If the response has only warnings, we still want to print output
+	if secret == nil || secret.IsOnlyWarnings() {
 		// Don't output anything unless using the "table" format
 		if Format(c.UI) == "table" {
 			c.UI.Info(fmt.Sprintf("Success! Data written to: %s", fullPath))
 		}
+
```

Comment:
This change is separate from `write.go`'s change because a kvv1 put do not return the same output as kvv2.